perception/probabilistic_occupancy_grid_map/lib/costmap_2d/occupancy_grid_map_projective.cpp

```diff
@@ -261,11 +261,10 @@ void OccupancyGridMapProjectiveBlindSpot::updateWithPointCloud(
   if (pub_debug_grid_) converter.addLayerFromCostmap2D(*this, "added_unknown", debug_grid_);
 
   // Third step: Overwrite occupied cell
-  for (size_t bin_index = 0; bin_index < obstacle_pointcloud_angle_bins.size(); ++bin_index) {
-    auto & obstacle_pointcloud_angle_bin = obstacle_pointcloud_angle_bins.at(bin_index);
+  for (auto & obstacle_pointcloud_angle_bin : obstacle_pointcloud_angle_bins) {
     for (size_t dist_index = 0; dist_index < obstacle_pointcloud_angle_bin.size(); ++dist_index) {
-      const auto & source = obstacle_pointcloud_angle_bin.at(dist_index);
-      setCellValue(source.wx, source.wy, cost_value::LETHAL_OBSTACLE);
+      const auto & outer_source = obstacle_pointcloud_angle_bin.at(dist_index);
+      setCellValue(outer_source.wx, outer_source.wy, cost_value::LETHAL_OBSTACLE);
```

Comment:
same comment
https://github.com/autowarefoundation/autoware.universe/pull/7942#discussion_r1672172760

Reply:
I had overlooked it. Sorry. I have corrected it.